python/addresources.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"github.com/jfrog/jfrog-client-go/utils/io/fileutils"
+	"io/ioutil"
+	"os"
+	"path"
+	"strings"
+)
+
+const (
+	pipDepTreeVersion             = "1"
+	pipDepTreeContentFilePrefix   = "pipDepTreeContent_v"
+	pipDepTreeContentRelativePath = "artifactory/utils/pip"
+	pipDepTreeContentFileName     = pipDepTreeContentFilePrefix + pipDepTreeVersion + ".go"
```

Comment:
I'm not sure that the new go source file created by *addresources.go* should have a long name, which also includes its version.
We can just name it *deptreescript.go*.
The *deptreescript.go* file actually includes the version as a variable, so please let me know if you need the version as the name fo the file.

Reply:
I wanted to be able to skip the file creation in case the file already exists with the latest version.
We can avoid adding the version to the file name if we will:
1. always write the file, override in case it's exist.
OR
2. read the version const from the file, and decide according to its value.

which way is preferred?